tools/generator/SourceWriters/Extensions/SourceWriterExtensions.cs

```diff
@@ -85,7 +85,7 @@ public static void AddInterfaceListenerEventsAndProperties (TypeWriter tw, Inter
 
 					AddInterfaceListenerEventsAndProperties (tw, iface, target, name, setter,
 						string.Format ("__v => {0} = __v", prop.Name),
-						string.Format ("__v => {0} = null", prop.Name), opt);
+						string.Format ("__v => {0} = null", prop.Name), opt, prop.Getter);
```

Comment:
The last `AddInterfaceListenerEventsAndProperties()` parameter name is `setMethod`, yet `prop.Getter` is used here.  Is this "mismatch" correct?

Reply:
I renamed it to `setListenerMethod` to help indicate it is not about property getters/setters, it is a reference to the method that is used to set the listener, like `addOnRoutingChangedListener`.  (Also added a comment in the code for future reference.)